### ISSUE TITLE
Deck Display Name uses Web Name instead of Deck Name

### ISSUE THREAD

**Issue by GiggliG** [Epix37/Hearthstone-Deck-Tracker#966]
I just noticed behaviour I think is new.

I often download decks from the web, and they have names like (made up example) "[Legend Top100] Axos Fantastic Token Druid", and then in HDT I rename them to something like "Axos Token".

Now, when I export such a deck to hearthstone, the name that gets displayed on the overlay, and the name that is entered in the hearthstone deck name, is the stupidly long one from the website, and not the name I have manually renamed.

Can this behaviour be corrected?

**Comment by GiggliG** [comment]
Since raising the issue above, I haven't seen this behaviour repeat. 

I had a deck where when i viewed it in hearthstone through the overlay, the displayed name was the wrong one. 

I went back to the tracker, opened the Edit Deck screen, saw the name was correct (the short name, not the web name). I then saved it again, and the name was now displayed properly in the overlay.

Not sure why this happened - maybe it was just a glitch with that single deck.
